packages/shared/components/AmountBox.svelte

```diff
@@ -8,12 +8,12 @@
     export let amount: number
     export let unit: string | undefined = undefined
 
-    const MAX_LENGTH_PER_LINE = 15
+    const MAX_LENGTH_PER_LINE: number = 15
```

Comment:
when there is type inference there is no need to type https://www.typescriptlang.org/docs/handbook/type-inference.html